src/material-experimental/mdc-slider/slider-thumb.ts

```diff
@@ -61,19 +61,19 @@ import {Thumb} from '@material/slider';
   /** Indicates which slider thumb this input corresponds to. */
   thumb: Thumb;
 
+  /** The hosts native HTML element. */
+  _hostElement: HTMLInputElement;
```

Comment:
Does this need to be its own property? you can just use `this._elementRef.nativeElement`

Reply:
It doesn't need to. I can also delete the `_hostElement` convenience variable in `slider-thumb.ts`